bin/docker-build

```diff
@@ -12,6 +12,7 @@ bindir=$( cd "${0%/*}" && pwd )
 "$bindir"/docker-build-proxy
 "$bindir"/docker-build-controller
 "$bindir"/docker-build-policy-controller
+"$bindir"/docker-build-policy-controller-gnu
```

Comment:
Remove the `"$bindir"/docker-build-policy-controller` entry because both `"$bindir"/docker-build-policy-controller` and `"$bindir"/docker-build-policy-controller-gnu` will target the same image:tag, so no point on calling the former. Also make sure to make the `"$bindir"/docker-build-policy-controller-gnu` file executable.